CadRevealFbxProvider/Attributes/ScaffoldingAttributeParser.cs

```diff
@@ -17,8 +17,18 @@ public class ScaffoldingAttributeParser
         ScaffoldingMetadata scaffoldingMetadata
     ) ParseAttributes(string[] fileLines)
     {
+        if (fileLines.Length == 0)
+            throw new ArgumentException(nameof(fileLines));
         Console.WriteLine("Reading attribute file.");
 
+        // The below will remove the first row in the CSV file, if it is not the header. However,
+        // this may be done directly from manipulation of the CsvOptions below, but since we have
+        // RowToSkip = 0, then SkipRow is never called.
```

Comment:
This comment is slightly misleading. 
```suggestion
        // The below will remove the first row in the CSV file, if it is not the header.
        // We tried using CsvReader SkipRow etc but they did not work for header rows.
```